templates/internet-of-things/partners.html

```diff
@@ -153,10 +153,9 @@ <h2 class="twelve-col">Commercial information</h2>
 {% endblock content %}
 {% block footer_extra %}
   <script>
-      YUI().use('event-base', function (Y) {
-          Y.on('domready', function () {
-              core.loadPartners('?programme__name=Internet%20of%20Things', '#iot-partners');
-          });
-      });
+      partnerLogoClouds = [{
```

Comment:
I'm guessing this ain't gonna work without `partners.js` being loaded afterwards.

In any case, let's move this to a function call instead.